packages/editor/src/data/data.ts

```diff
@@ -25,14 +26,17 @@ const findComponent = (
   if (id.startsWith(TABLE_DROPZONE_ID_PREFIX)) {
     return findTableComponent(data, id.replace(TABLE_DROPZONE_ID_PREFIX, ''));
   }
+  if (id.startsWith(FIELDSET_DROPZONE_ID_PREFIX)) {
+    return findFieldsetComponent(data, id.replace(FIELDSET_DROPZONE_ID_PREFIX, ''));
+  }
```

Comment:
then also this is no longer needed -> merge with layout dropzone id prefix to structure whatever prefix